test/cpp/end2end/xds/xds_outlier_detection_end2end_test.cc

```diff
@@ -194,7 +199,8 @@ TEST_P(OutlierDetectionTest, SuccessRateMaxPercent) {
                           .set_server_expected_error(StatusCode::CANCELLED));
   CheckRpcSendOk(DEBUG_LOCATION, 1, rpc_options2);
   CheckRpcSendOk(DEBUG_LOCATION, 1, rpc_options3);
-  gpr_sleep_until(grpc_timeout_milliseconds_to_deadline(100));
+  gpr_sleep_until(grpc_timeout_milliseconds_to_deadline(
+      10 * get_test_interval() / absl::Milliseconds(1)));
```

Comment:
Do we need to actually sleep for 10x the interval in all of these tests in order to make them pass reliably?  If it's possible to use a lower multiplier without introducing flakiness, that seems preferable, since it will allow the tests to finish faster.

Reply:
I did not have time to test individual tests with different values but x10 was the first multiplier that passed after ~50k runs with 100% after trying x2 and x4. Let me try x4~x8 again and update.

Reply:
Done.